batteries/dq_dv.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jan  5 10:10:36 2023
+
+@author: mpopeil
+"""
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import sklearn as sk
+import sklearn.datasets
+from pandas import DataFrame
+from scipy.interpolate import Akima1DInterpolator
```

Comment:
For long named methods like this, you can give it a shorter name, which makes your code a little more concise:

```suggestion
from scipy.interpolate import Akima1DInterpolator as akima
```

Reply:
Lol, just noticed that we don't actually use this interpolator. I guess just delete this line, then.